### PR TITLE
Setup applyconfiguration-gen with aqua

### PR DESCRIPTION
**Pull request by zoetrope**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -115,6 +115,7 @@ release-build: setup
 
 setup:
 	aqua i -l
+	aqua policy allow ./aqua-policy.yaml
```
Comment (ymmt2005) [inline Makefile:118]: Not sure, but doesn't this need to come before `aqua i -l`?

Comment (zoetrope) [reply Makefile:118]: Thanks!
That's right. I don't know why but CI passed 🤔

Comment (ymmt2005) [reply Makefile:117]: @zoetrope 
My guess is:
1. `aqua -i l` installs only links to a hook script.
2. When the CI actually runs a command installed by Aqua, the hook script tries to install the real command.
3. At that time, `aqua policy allow` has been run, so the hook script succeeds.

Comment (ymmt2005) [reply Makefile:117]: Also note that we use `aqua-installer` action that runs `aqua -i l` by default.
https://aquaproj.github.io/docs/products/aqua-installer/#optional-inputs

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**